common/app/model/content.scala

```diff
@@ -386,7 +389,8 @@ object Content {
           .orElse(elements.mainPicture.flatMap(_.images.largestImageUrl))
           .orElse(trail.trailPicture.flatMap(_.largestImageUrl))
           .getOrElse(Configuration.images.fallbackLogo)
-      }
+      },
+      blocks = apiContent.blocks
```

Comment:
for live blogs I converted the low level capi blocks into something more useful for live blogs here: https://github.com/guardian/frontend/blob/155f738f5b9e22cf8c894bc44527966e630a10cf/common/app/model/liveblog/BodyBlock.scala#L13

I'm not sure what the usual practice is regarding carring over capi models into the high level model, I think normally we rejig into a condensed format for rendering at this point.  Maybe canvas opinion on that one a bit.